config.php

```diff
@@ -3,7 +3,7 @@
 #General Informations
 $CONFIG_CONTACT = 'fsi@fsi.uni-tuebingen.de';
 $CONFIG_TERM = 'WS 20/21'; # example: WS 19/20 or SS 20
-$fp = "../../../eei-registration/"; #File Prefix
+$fp = "../../eei-registration/"; #File Prefix
```

Comment:
Just a heads-up: This seems a bit hacky (I assume this only works if `config.php` gets included from a script at the next (or 2nd?) directory level below). Wouldn't something like `__DIR__/../eei-registration` be better (see https://www.php.net/manual/en/language.constants.predefined.php).